pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1731,6 +1731,122 @@ func runMessageTooLarge(ctx context.Context, t test.Test, c cluster.Cluster) {
 	require.Regexp(t, `mvcc=[\d\.]+,\d+`, logStr, "log should include mvcc")
 }
 
+type multiTablePTSBenchmarkParams struct {
+	numRows   int
+	numTables int
+	duration  string
+}
+
+// runCDCMultiTablePTSBenchmark is a benchmark for changefeeds with multiple tables,
+// focusing on the performance of the PTS system. It starts a bank workload on every
+// table it creates and then runs a single changefeed that targets all of these bank tables.
+// Each of those workloads (there will be one per table) will run for the duration specified
+// in the params and have the number of rows specified in the params.
+func runCDCMultiTablePTSBenchmark(
+	ctx context.Context, t test.Test, c cluster.Cluster, params multiTablePTSBenchmarkParams,
+) {
+	ct := newCDCTester(ctx, t, c)
+	defer ct.Close()
+
+	startOpts := option.DefaultStartOpts()
+	startOpts.RoachprodOpts.ExtraArgs = append(startOpts.RoachprodOpts.ExtraArgs,
+		"--vmodule=changefeed=2",
+		"--vmodule=changefeed_processors=2",
+		"--vmodule=protected_timestamps=2",
```

Comment:
I suspect this won't do what you want it to do, i.e. the last value of the flag will be used. You can instead pass all of them as a comma-separated list with a single flag.